Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml

```diff
@@ -328,6 +334,24 @@
     spawned:
     - id: FoodMeatRouny
       amount: 3
+      
+- type: entity
+  name: Friendshaped
+  parent: MobXenoRunnerNPC # Delta V - These no longer inherit the ghost takeover comp
+  id: MobFriendshapedNPC # Delta V - These no longer inherit the ghost takeover comp
+  components:
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Aliens/Xenos/rouny.rsi
+    offset: 0,0.6
+  - type: NpcFactionMember
+    factions:
+    - PetsNT
+  - type: Butcherable
+    butcheringType: Spike
+    spawned:
+    - id: FoodMeatRouny
+      amount: 3
```

Comment:
```suggestion
- type: entity
  name: Friendshaped
  parent: MobXenoRunnerNPC
  id: MobFriendshapedNPC
  components:
  - type: NpcFactionMember
    factions:
    - PetsNT
  - type: Butcherable
    butcheringType: Spike
    spawned:
    - id: FoodMeatRouny
      amount: 3
```

Reply:
> 

A note to you, The current Rouny sprite in game runs off of the same parameters for the offset, so removing it as I have done will make the two sprites actually be different as to their placement in game.

Reply:
the ent is currently parented to the other rouny meaning its will use the same sprite as the orignal rouny

Reply:
alright, so why am I getting an error